cumulusci.yml

```diff
@@ -232,6 +232,12 @@ flows:
     config_managed:
         # description: Configure an org for use as a dev org after package metadata is deployed.
         steps:
+            2:
+                task: update_admin_profile
+                options:
+                    managed: True
```

Comment:
In theory this shouldn't be needed with CumulusCI 3.23.0 and higher - it knows how to infer the `managed` and `namespaced_org` options. Are you seeing behavior that indicates otherwise?